src/config/constants/swapWarningTokens.ts

```diff
@@ -0,0 +1,18 @@
+const SwapWarningTokens = {
+  safemoon: {
+    symbol: 'SAFEMOON',
+    address: {
+      56: '0x8076C74C5e3F5852037F31Ff0093Eeb8c8ADd8D3',
+      97: '',
+    },
+  },
+  bondly: {
+    symbol: 'BONDLY',
+    address: {
+      56: '0x96058f8C3e16576D9BD68766f3836d9A33158f89',
+      97: '',
+    },
+  },
+}
+
+export default SwapWarningTokens
```

Comment:
Is possible to avoid keeping a list like this? For example maybe keeping a list of just the symbol and get the address from a static list somewhere (to avoid maintaining another list)?

Reply:
Good point man, I understand the reservations about _another_ list. Part of the spec for this was to use addresses rather than symbols when matching selected token with warning token. So while it would be lighter on the config - I figured an array of uncontextualised addresses would also be different to manage.

I could certainly import `bondly` from our `tokens.ts`, but for some - like safemoon in this example, they aren't guaranteed to exist elsewhere in the codebase.

Reply:
Levelled up the config, see what you think.